pkg/forwarder/forwarder.go

```diff
@@ -562,20 +568,22 @@ func (f *DefaultForwarder) SubmitContainerLifecycleEvents(payload Payloads, extr
 	return f.sendHTTPTransactions(transactions)
 }
 
-func (f *DefaultForwarder) submitProcessLikePayload(ep transaction.Endpoint, payload Payloads, extra http.Header, retryable bool) (chan Response, error) {
+func (f *DefaultForwarder) submitProcessLikePayload(ep transaction.Endpoint, payload Payloads, extra http.Header, retryable bool, attempts bool) (chan Response, error) {
```

Comment:
Do we want to set this header `X-DD-Agent-Attempts` for manifest ?

Reply:
More information is probably always good, but that is definitely something we only have in the new endpoint to parse, not on the other side (our)